src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/CftQueryServiceITTest.java

```diff
@@ -88,19 +88,24 @@ void setUp() {
         "paginatedResultsScenario"
     })
     void shouldRetrieveTasks(TaskQueryScenario scenario) {
+
+        System.out.println(scenario.scenarioName);
```

Comment:
we can remove this System,out.

Reply:
Since this does not do any harm at the moment I will remove as part of the subsequent PR which follows this one https://tools.hmcts.net/jira/browse/RWA-1133